examples/1-getting-started.ipynb

```diff
@@ -0,0 +1,322 @@
+    "... or you can just **wrap you favorite Scikit-learn estimator** like we do here:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "from sklearn.externals import joblib\n",
+    "\n",
+    "class RandomForestTransformer(BaseTransformer):\n",
+    "    def __init__(self):\n",
+    "        self.estimator = RandomForestClassifier()\n",
+    "        \n",
+    "    def fit(self, X, y):\n",
+    "        self.estimator.fit(X, y)\n",
+    "        return self\n",
+    "\n",
+    "    def transform(self, X, **kwargs):\n",
+    "        y_pred  = self.estimator.predict(X)\n",
+    "        return {'y_pred': y_pred}  # TODO: exaplain this\n",
+    "    \n",
+    "    def save(self, filepath):\n",
+    "        joblib.dump(self.estimator, filepath)\n",
+    "        \n",
+    "    def load(self, filepath):\n",
+    "        self.estimator = joblib.load(filepath)\n",
+    "        return self"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So what does the transformer do? It must be able to:\n",
+    "* **initialize** itself\n",
+    "* **fit** and **transform** the incoming data prepared by the adapter; when transforming, the result should be returned as a **dictionary** that can be **passed on to the next step**\n",
+    "* **save** and **load** its parameters; this is handy when you're trying to avoid re-computing things over and over.\n",
+    "\n",
+    "See how flexible this is? You can just as easily wrap your Keras or Pytorch models."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's combine our adapter and transformer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "classifier_step = Step(name='classifier',\n",
+    "                       transformer=RandomForestTransformer(),\n",
+    "                       input_data=['input'],                 \n",
+    "                       adapter=input_adapter,\n",
+    "                       cache_dirpath='./cache'\n",
+    "                      )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And that's our one-step pipeline finished. You can visualize it too:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "classifier_step"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is just about the simplest pipeline you can imagine. Now let's train it!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Training"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "classifier_step.clean_cache()\n",
```

Comment:
@mromaniukcdl I wouldn't introduce `.clean_cache()` just yet. Let's leave this for the future (notebook explaining `cache_output`, `save_output` and stuff)

Reply:
+1 in principle but I think we would like to turn off caching altogether in this introductory tutorial